locales/en-US/messages.properties

```diff
@@ -39,6 +39,9 @@ errorUnsupportedBrowserSuggestion=Thimble works in all of the following browsers
 errorUnsupportedBrowserIgnore=Let me try anyway!
 errorLoadingProjectBramble=For error details open your browser's Developer Tools console.
 refreshForUpdates=<strong>Thimble has Updates</strong> - Please refresh your browser to get the latest changes. <button class="refresh-browser">Refresh</button>
+projectNotLoadingMessage=If your project doesn't load, <br>please reload the page to try again.
```

Comment:
What about `Your project seems to be taking a long time to load. If it doesn't load, please reload the page to try again.`